pkg/scrape/lethalhardcorevr.go

```diff
@@ -0,0 +1,138 @@
+package scrape
+
+import (
+  "log"
+  "regexp"
+  "strings"
+
+  "github.com/gocolly/colly"
+  "github.com/mozillazg/go-slugify"
+  "github.com/thoas/go-funk"
+)
+
+func isGoodTag(lookup string) bool {
```

Comment:
This is interesting approach - there's no prefilter in any other scraper, but it probably should be!

Reply:
I added it because the quality of the tags is horrible... and they're really HTTP meta keywords for SEO with scene tags stuffed in.

This site appears to be affiliated with adultempire somehow and the videos are iframes to adultempire.com. There was actually a little more useful data in regular DOM tags in the iframe but for simplicity I decided to parse the data from the scene page instead.